archivist/src/main/scala/uk/ac/wellcome/platform/archive/archivist/bag/ArchiveItemJobCreator.scala

```diff
@@ -22,17 +21,17 @@ object ArchiveItemJobCreator {
     * If any of the manifests are incorrectly formatted, it returns an error.
     *
     */
-  def createArchiveDigestItemJobs(job: ArchiveJob)
-    : Either[ArchiveError[ArchiveJob], List[ArchiveDigestItemJob]] =
+  def createArchiveDigestItemJobs(
```

Comment:
nit: drop the word "Archive" from this method?

```suggestion
  def createDigestItemJobs(
```